test/spinxsk/spinxsk.c

```diff
@@ -1992,6 +2025,97 @@ ProcessPkts(
     return TRUE;
 }
 
+#define FNMP_FRAME_MAX_BACKFILL 64
```

Comment:
rather than copying this into yet another project, i think it's time we resolve https://github.com/microsoft/win-net-test/issues/30 and consume it here